contracts/ERC721R.sol

```diff
@@ -367,7 +380,10 @@ contract ERC721r is Context, ERC165, IERC721, IERC721Metadata {
         require(ERC721r.ownerOf(tokenId) == from, "ERC721: transfer from incorrect owner");
         require(to != address(0), "ERC721: transfer to the zero address");
 
-        _beforeTokenTransfer(from, to, tokenId);
+        uint256[] memory tokenIds = new uint256[](1);
+        tokenIds[0] = tokenId;
+
+        _beforeTokenTransfer(from, to, tokenId, tokenIds);
```

Comment:
Can you use a literal here? eg `[2]`?

Reply:
Or I guess your approach seems to be standard. From OZ 1155:

<img width="972" alt="image" src="https://user-images.githubusercontent.com/59190/180655230-354f60a9-8682-4f0e-bfa3-cbf78d55dce5.png">